doc/source/serve/autoscaling.md

```diff
@@ -0,0 +1,43 @@
+(serve-autoscaling)=
+
+# Serve Autoscaling
+
+This section should help you:
+
+- Understand how Ray Serve autoscaling works
+- Learn how to configure the parameters for your workload
+
+
+## Autoscaling architecture
+Ray Serve autoscaling feature automatically increases or decreases the number of replicas based on the load. 
+
+![pic](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling.svg)
```

Comment:
Nice diagram! This will be very helpful when explaining how Ray Serve's autoscaling works.

<img width="854" alt="Screen Shot 2022-06-13 at 3 55 22 PM" src="https://user-images.githubusercontent.com/92341594/173460751-b2d155c4-aff6-4fc7-b132-b5b6799dd6aa.png">

A few suggestions:

1. Can we rename the "node" in the controller to "Ray Head Node"? And similarly, can we name the other nodes to "Ray Worker Node? This is mainly to distinguish between Ray nodes (which are actual machines) and Serve deployment nodes (which are used in deployment graphs).

2. In each Ray node, we have two "worker replica" rectangles. The left rectangle has no outline, but the right rectangle has an outline. Can we remove the outlines on the right rectangles for consistency?

<img width="277" alt="Screen Shot 2022-06-13 at 3 59 33 PM" src="https://user-images.githubusercontent.com/92341594/173461345-bb4a8b9f-d0e7-44f1-a251-c5de8fca14c8.png">

3. There's a typo in "contoller." It should be "controller."

4. There's a black diagonal "notch" under the controller node. Can we remove it? Can we also move the word "node" in that rectangle, so it doesn't overlap with the arrow?

<img width="300" alt="Screen Shot 2022-06-13 at 4 03 02 PM" src="https://user-images.githubusercontent.com/92341594/173461599-eaa6222d-1024-4e19-96a1-13e57efdb617.png">

5. Can we make the legend left-aligned?

6. The yellow arrow is a bit tough to see. Could we change it (and the corresponding worker replica) to a slightly darker color?

7. To maintain consistency with the rest of the docs, could we change "client (deployment handle)" to "[deployment's `ServeHandle`](https://docs.ray.io/en/releases-1.13.0/serve/http-servehandle.html#servehandle-calling-deployments-from-python)?"

8. Could we add a section to the diagram explaining how HTTP requests fit into this? We could add one more box to the right of the diagram representing an HTTP request, and we could draw an arrow from it to one of the `ServeHandles`.